src/stateFromMarkdown.js

```diff
@@ -1,11 +1,25 @@
 /* @flow */
 
-import MarkdownParser from './MarkdownParser';
 import {stateFromElement} from 'draft-js-import-element';
+import remark from 'remark';
+import html from 'remark-html';
+import {jsdom} from 'jsdom';
```

Comment:
I'm hesitant to pull in `jsdom` dependency since the browser natively does everything that jsdom does. But this is an interesting approach to go `markdown -> ast -> html-string -> dom -> contentState`. I'm curious to know what the downsides, if any are to double-parsing.